_tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md

```diff
@@ -1,14 +1,17 @@
 ---
 layout: default
-title: Snapshot Management API
+title: Snapshot management API
 parent: Snapshots
 nav_order: 30
 has_children: false
+grand_parent: Availability and Recovery
+redirect_from: 
+  - /opensearch/snapshots/sm-api/
 ---
 
 # Snapshot Management API
 
-Use the [Snapshot Management (SM)]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) API to automate [taking snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots). 
+Use the [snapshot management (SM)]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) API to automate [taking snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots). 
```

Comment:
Why do two separate links in the same sentence take you to the same place?

Reply:
I'll remove the first.